benchalerts/parse_conbench.py

```diff
@@ -24,6 +24,40 @@ def _clean(text: str) -> str:
     return textwrap.fill(textwrap.dedent(text), 10000).replace("  ", "\n\n").strip()
 
 
+def benchmarks_with_errors(
+    comparisons: List[RunComparison],
+) -> List[Tuple[str, str, str, str]]:
+    """Find the run IDs, webapp links, display names, and error messages of benchmark
+    cases that had errors."""
+    out = []
+
+    for comparison in comparisons:
+        if comparison.compare_results:
+            out += [
+                (
+                    comparison.contender_id,
+                    comparison.compare_link,
+                    case["benchmark"],
+                    case["contender_error"]["message"],
+                )
+                for case in comparison.compare_results
+                if case["contender_error"]
+            ]
+        else:
+            out += [
+                (
+                    comparison.contender_id,
+                    comparison.contender_link,
+                    case["tags"].get("name", str(case["tags"])),
```

Comment:
Right now the GET /benchmarks endpoint doesn't add a display name like the GET /compare endpoint does. I'll make an issue in conbench so that these names can be standard among the two implementations.

Reply:
Thanks for creating that issue! Yeah there are a few places like this that would be lovely to clean up

Reply:
https://github.com/conbench/conbench/issues/410